packages/smooth_app/lib/cache/files/files_cache_impl.dart

```diff
@@ -0,0 +1,121 @@
+// ignore_for_file: avoid_classes_with_only_static_members
```

Comment:
It's never the case here, is it?

Reply:
Yes, it was only during my devs.
Removing it